src/prometheus.mli

```diff
@@ -113,6 +114,9 @@ module Counter : sig
   val inc_one : t -> unit
   val inc : t -> float -> unit
   (** [inc t v] increases [t] by [v], which must be non-negative. *)
+
+  val get : t -> float
+  (** [get t] returns the current value of the counter. *)
```

Comment:
Why are we exposing this?

Reply:
In our use-case we'd like to use the accumulated values for internal decision-making and posting some of the values over another interface towards another system. To do that we want to read back the embedded values.